prisma/schema.sql

```diff
@@ -40,21 +37,42 @@ CREATE TABLE verification_requests (
   created_at TIMESTAMPTZ NOT NULL DEFAULT CURRENT_TIMESTAMP,
   updated_at TIMESTAMPTZ NOT NULL DEFAULT CURRENT_TIMESTAMP
 );
-
 -- create program table
 CREATE TABLE programs (
   id SERIAL PRIMARY KEY NOT NULL,
   price INTEGER NOT NULL, -- price in cents, to make it integer
+  online_format online_formats NOT NULL,
+  tag TEXT NOT NULL, -- art, music, math etc (TODO we might need a tag table later)
+
+  start_date TIMESTAMPTZ NOT NULL, -- earliest start date of all sections
+  end_date TIMESTAMPTZ NOT NULL, -- latest end date of all sections
+
+  is_archived BOOLEAN DEFAULT false NOT NULL,
+  created_at TIMESTAMPTZ NOT NULL DEFAULT CURRENT_TIMESTAMP,
+  updated_at TIMESTAMPTZ NOT NULL DEFAULT CURRENT_TIMESTAMP
+);
+
+CREATE TABLE sections (
```

Comment:
We should have `ON DELETE CASCADE` so that if a program gets deleted, the associating records here also get deleted.